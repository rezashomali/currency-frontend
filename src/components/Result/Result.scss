.result {
  display: flex;
  align-items: flex-end;

  p {
    font-size: 30px;
  }
}
