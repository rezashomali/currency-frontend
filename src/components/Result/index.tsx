import React from "react";
import "./Result.scss";
interface Props {
  data: number;
}

const Result: React.FC<Props> = ({ data }) => {
  return (
    <div className="result">
      <span>$</span>
      <p> {data} </p>
    </div>
  );
};

export default Result;
