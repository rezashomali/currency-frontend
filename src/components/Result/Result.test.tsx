import React from "react";
import renderer from "react-test-renderer";
import Result from "./index";

test("Result snapshot ", () => {
  const component = renderer.create(<Result symbol="$" data={0} />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
