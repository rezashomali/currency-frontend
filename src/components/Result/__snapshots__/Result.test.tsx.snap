// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Result snapshot  1`] = `
<div
  className="result"
>
  <p>
    <span>
      $
    </span>
     
    0
  </p>
</div>
`;
